ruby/vampires.rb

```diff
@@ -0,0 +1,55 @@
+
+puts "How many employees will be processed?"
+employees = gets.chomp
+
+employees_loop = 0
+
+until employees.to_i == employees_loop
+	employees_loop +=1
+
+	puts "What is your name?"
+	vamp_name = gets.chomp
+
+	puts "How old are you?"
+	age = gets.chomp
+
+	puts "What year were you born?"
+	birth_year = gets.chomp
+
+	puts "Our company cafeteria serves garlic bread. Should we order some for you? (y/n)"
+	garlic_bread = gets.chomp
+
+	puts "Would you like to enroll in the company’s health insurance? y/n"
+	health_insurance = gets.chomp
+
+	puts "Do you have any allergies? Type 'done' when finished."
+	allergies = 0
+	loop do
+		allergies = gets.chomp	
+		if allergies == "sunshine"	
+			vampire = "Probably a vampire."
+			break
+		elsif allergies == "done" 
+			break
+		end
+	end	
+
+	if age.to_i == 2017-birth_year.to_i && (garlic_bread == "y" || health_insurance == "y")
```

Comment:
fair amount of repetition, with things like  age.to_i == 2017-birth_year.to_i so consider extracting these concepts into 

correct_age = age.to_i == 2017-birth_year.to_i
likes_garlic = garlic_bread == "y"
and so forth